src/ansys/fluent/core/fluent_connection.py

```diff
@@ -225,9 +225,12 @@ def _get_channel(ip: str, port: int):
     )
 
 
-class _ConnectionProperties:
-    def __init__(self, scheme_eval):
-        self.scheme_eval = scheme_eval
+class _ConnectionInterface:
```

Comment:
@prmukherj rename `fluent_build_info()` -> `product_build_info()`?

Reply:
Done, Thank you.